browsers/samsunginternet_android.json

```diff
@@ -161,13 +161,13 @@
         },
         "11.2": {
           "release_date": "2020-03-22",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "75"
         },
         "12.0": {
           "release_date": "2020-05-07",
```

Comment:
Ordinarily, you'd be right about not changing the date, but I think the date was left in place erroneously. 
We probably should've removed it—or checked in with Samsung about an accurate date—in the course of #6219.
```suggestion
          "release_date": "2020-06-19",
```